### PR TITLE
Product UI complete

### PR DESCRIPTION
**Pull request by gamerpizza**
1) I made 2 components for rating and quantity because they are reused.
2) I made a page for displaying a single product. 
3) Products that are displayed at the end of the page are rendered from API. I will change the logic to display products of the same category as the product page once the API for displaying a single product is available.

### CODE REVIEW

File: components/Products/Products.tsx [hunk 1]
```diff
@@ -5,18 +5,19 @@ import startIcon from "../../public/images/star.svg";
 import addIcon from "../../public/images/add.svg";
 
 const Products = (props: any) => {
-  console.log(props.props);
```
Comment (rogelioConsejo) [inline components/Products/Products.tsx:8]: It's always nice to see someone cleaning old code 💎

File: pages/_app.tsx [hunk 2]
```diff
@@ -88,7 +88,6 @@ function MyApp({ Component, pageProps, categories }: any) {
       });
     }
   };
-  console.log("pageprops", pageProps);
```
Comment (rogelioConsejo) [inline pages/_app.tsx:91]: I think I am going to love working with you 🙂

File: pages/cart/index.tsx [hunk 3]
```diff
@@ -57,33 +57,8 @@ const Cart = (props: any) => {
...
-                                    className="font-medium text-black w-5 h-5 bg-[#F5F8FA] hover:text-indigo-500"
-                                  >
-                                    +
-                                  </button>
-                                </div>
-
-                                <p className="text-[#1C1F22] font-comfortaa">
-                                  {props.cart[item].quantity}
-                                </p>
+                              <Quantity addToCart= {() => props.addToCart(item, "", 1, 249, "usman")} removeFromCart={() => props.removeFromCart(item)} quantity={props.cart[item].quantity}/>
```
Comment (rogelioConsejo) [inline pages/cart/index.tsx:60]: Cool!

**Review by rogelioConsejo — APPROVED** [review]

File: pages/_app.tsx (hunk 2 above)
Comment (gamerpizza) [reply pages/_app.tsx:91]: I am going to love it more :smile:

**Pull request merged**